CHANGELOG.md

```diff
@@ -16,6 +21,11 @@ _2023-03-30_
 _2023-03-29_
 
 <!-- Links -->
-[Unreleased]: https://github.com/adevinta/spark-android/compare/0.0.2...HEAD
+
+[Unreleased]: https://github.com/adevinta/spark-android/compare/0.0.3...HEAD
+
+[0.0.2]: https://github.com/adevinta/spark-android/releases/tag/0.0.3
+
```

Comment:
These new lines seems unnecessary. Is it because of auto-formatting on your end?

Reply:
Ah yes I have auto reformat enabled